docs/java-helloworld.md

```diff
@@ -0,0 +1,145 @@
+---
+id: java-helloworld
+title: Java SDK "Hello World!" tutorial
+sidebar_label: Java "Hello World!"
+tags: helloworld, java, sdk, intellij, gradle, tutorial
+---
+
+This tutorial is aimed at developers who are new to [Temporal](/docs/overview), beginner Java programmers, or both. It will teach you how to build a simple "Hellow World!" Temporal Workflow application using the [Temporal Java SDK](https://github.com/temporalio/java-sdk) as well as a few basics, such as:
```

Comment:
I would say that it assumes basic knowledge of Java programming. I don't think someone who has no clue about Java is the target audience.